pkg/pullrequest/e2e/e2e_test.go

```diff
@@ -0,0 +1,155 @@
+package e2e
```

Comment:
Should these files be in test/ package/folder like the rest of our e2e tests?

Reply:
I would definitely prefer it to be in `test/pullrequest` or something indeed :angel: